### DISCUSSION TITLE
How do I regenerate my changelogs from scratch?

### DISCUSSION THREAD

**Discussion by JadedBlueEyes** [release-plz/release-plz#1558] [Q&A] [answered]
The changelogs in my project have become slightly inconsistent over time with changes to my git-cliff configuration. How do I regenerate them from scratch using my current configuration? I've tried invoking git-cliff directly, but it produces a changelog for the repository as a whole rather than individual projects.

**Comment by marcoieni** [comment, marked as answer]
It's not possible at the moment. https://github.com/MarcoIeni/release-plz/issues/1009

Pr welcome!
